HULKs/hulk
# Add a progress indicator to pepsit

## Introduced Changes

This PR adds a progress indicator to pepsi's command line output.

## ToDo / Known Issues

## Ideas for Next Iterations (Not This PR)

## How to Test

Run the affected pepsi commands and observe their output.
Try running the commands on a different number of naos,
or introduce errors by attempting to execute commands on naos
which are unavailable.
